test/manifold/executor_test.clj

```diff
@@ -58,8 +58,8 @@
   (let [tf (thread-factory)]
     (.newThread tf (constantly nil)))
   (let [tf (thread-factory
-            (fn [group target _]
-              (proxy [Thread] [group target "custom-name"])))
+            (fn [group target _ stack-size]
+              (proxy [Thread] [group target "custom-name" stack-size])))
```

Comment:
What's `proxy` for? Couldn't we call the Thread constructor? 

If the point is to demonstrate how a user might override the default behavior, might it still be preferable to make a fn that calls the Thread constructor, and then demonstrate using the Thread methods to alter it, to serve as a better example?

It's not a big deal, I'm just confused as to the purpose as it currently stands.

Reply:
I guess I was overthinking there.
The aim was to prove we can pass a subclass of a `java.lang.Thread`.
Anyway, just building a new Thread is probably enough.